cmake/modules/RootMacros.cmake

```diff
@@ -1539,8 +1539,8 @@ endfunction()
 function(ROOT_ADD_PYUNITTESTS name)
   set(ROOT_ENV ROOTSYS=${ROOTSYS}
       PATH=${ROOTSYS}/bin:$ENV{PATH}
-      LD_LIBRARY_PATH=${ROOTSYS}/lib:$ENV{LD_LIBRARY_PATH}
-      PYTHONPATH=${ROOTSYS}/lib:$ENV{PYTHONPATH})
+      LD_LIBRARY_PATH=${ROOTSYS}/lib:${ROOTSYS}/lib/${python_dir}:$ENV{LD_LIBRARY_PATH}
```

Comment:
Check if is PyROOT experimental and only if true, add `${ROOTSYS}/lib/${python_dir}`